content/docs/addons-shallow-renderer.md

```diff
@@ -30,7 +30,7 @@ function MyComponent() {
 }
 ```
 
-Then you can assert:
+तब आप जांच कर सकते हैं:
```

Comment:
assert has a specific meaning to it, we should just convert it to hindi

Reply:
What should we use? असर्ट or अस्सर्ट

Reply:
अस्सर्ट makes more sense in my opinion, thoughts @arshadkazmi42 ?
Also, whatever is decided, please do add it in #2 as well

Reply:
I was also thinking about this `अस्सर्ट`